src/test/resources/reportDetailsTest3.json

```diff
@@ -161,7 +161,7 @@
       },
       {
         "value": {
-          "hearingDate": "15 October 2020",
```

Comment:
Shouldn't any date format be accepted and then it be changed to the correct format?

Reply:
The date "15 October 2020" that was previously in the unit test is wrong. The value needs to reflect what is stored in CCD which is "hearingDate": "2020-10-12T10:00:00.000".
When the CCD data is sent to docmosis then it gets formatted from the CCD format to a user friendly format to appear on the report

Reply:
ok cool, if it would be same date format from ccd, then all good :)